chat-plugins/nightclub.js

```diff
@@ -0,0 +1,154 @@
+//Nightclub by SilverTactic (Siiilver) and panpawn
+'use strict';
+
+if (!global.nightclubs) global.nightclubs = {};
```

Comment:
uhh I'll honestly leave that to you and silveee, panpawn because it's your guys plugin but I know Kanabae wants this plugin as do a few of the other marketplace staff - It's a pretty cool one. 

- I say I'm leaving it because I don't wanna mess with this and end up breaking it xaa